style.css

```diff
@@ -0,0 +1,115 @@
+
+body {
+    text-align: center;
+    width: 100%;
+    line-height: 3em;
+}
+
+#keyboard {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    border: 2px solid #6292c2;;
+    padding: 5px;
+    max-width: 900px;
+    margin: 0 auto;
+    text-align: center;
+    position: relative;
+}
+
+#keyboard::after {
+    content: "";
+    display: block;
+    clear: both;
+}
+
+.k-key {
+    width: 50px;
+    height: 50px;
+    border: 1px solid black;
+    margin: 3px;
+    padding: 3px;
+    cursor: pointer;
+    background: #6292c2;
+    border: 2px solid #eee;
+    -webkit-border-radius: 50px;
+    -moz-border-radius: 50px;
+    border-radius: 15px;
+    /* transition: All 1s ease; */
+    text-decoration: none;
+  outline: none;
```

Comment:
Different space setting?

Reply:
I've just corrected it. I used the same indent everywhere